README.md

```diff
@@ -11,8 +11,16 @@ This repository contains the [OneTrust](https://www.onetrust.com) integration fo
     ```groovy
     dependencies {
         implementation 'com.mparticle:android-onetrust-kit:5+'
+    // Implement the SDK version that corresponds to the published version you're using'
+        implementation 'com.onetrust.cmp:native-sdk:X.X.0.0'
```

Comment:
```suggestion
        // Implement the SDK version that corresponds to the published version you're using'
        implementation 'com.onetrust.cmp:native-sdk:X.X.0.0'
```

Reply:
just fixing indentation here. but this is a good change to add to the README, which was already necessary before due to us using compileOnly